tests/generated_files/under_version_control/genes/test_gene_with_targets.fasta

```diff
@@ -0,0 +1,3 @@
+>gene_with_targets|flash_resistance:diaminopyrimidine_antibiotic,diaminopyrimidine_antibiotic__trimethoprim|flash_key:dfrB1__AY139601__ARO_3002864|flash_aro:3002864
```

Comment:
Is it the case that the cut sites in dna_good_structure are the ones generated for this gene? If not, it might be nice to make a test gene for which those are CRISPR targets. (If they are a subset of the targets, then one could remove the rest by replacing them with AAAAAAA.)

Reply:
This is a fake/test gene. I'll use one of the real genes that is smaller so that it reflects a real situation and the targets are genuine targets. That'll make it easier to re-use for other tests as well.